### DISCUSSION TITLE
Primitive Obsession pode ser uma dor de cabeça?

### DISCUSSION THREAD

**Discussion by lrodolfol** [balta-io/forum#275] [Tira Dúvidas] [answered]
Boa tarde pessoal. falando ai de Primitive Obsession. Se eu ficar criando tipos complexos no projeto, isso não impactaria na hora de relacionar no banco de dados?
ai um usuário vai ter relação com ad classes:     Nome, Endereço, Documento, Email e TipoPessoa

só nesse exemplo, quando eu for usar o ef com GET de usuários, eu vou precisar fazer 5 include para trazer as informações.
Imagina

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @lrodolfol, como vai?

Na verdade, usar objetos de valor não afeta a forma como você modela o banco. Você pode ter VOs e mapeá-los para um *tabelão* por exemplo.
